src/include/units/bits/common_quantity.h

```diff
@@ -56,8 +56,8 @@ struct common_quantity_impl<quantity<D, U1, Rep1>, quantity<D, U2, Rep2>, Rep> {
 };
 
 template<typename D1, typename U1, typename Rep1, typename D2, typename U2, typename Rep2, typename Rep>
-  requires same_unit_reference<dimension_unit<D1>, dimension_unit<D2>>::value
-struct common_quantity_impl<quantity<D1, U1, Rep1>, quantity<D2, U2, Rep2>, Rep> {
+requires same_unit_reference<dimension_unit<D1>, dimension_unit<D2>>::value struct common_quantity_impl<
+    quantity<D1, U1, Rep1>, quantity<D2, U2, Rep2>, Rep> {
```

Comment:
Please reformat as it was before. clang-format still does not work as expected.

Reply:
Will do.